doc/dnf5.conf.5.rst

```diff
@@ -1358,6 +1358,15 @@ Variables Directories
 ``Distribution Variables Directory``
     /usr/share/dnf5/vars.d/
 
+
+Source and debuginfo repository names
```

Comment:
I'd like to see this piece of information to be close to where repository file format is documented. We don't have a dedicated page for it. But we have "Repo Options" and "Repo Variables" sections in this page. Could you move this section between the two sections?